tests/integration/standalone/alice-home-init.nix

```diff
@@ -1,5 +1,3 @@
-{ config, pkgs, ... }:
-
```

Comment:
bf893ad4 Robert Helgesson (2025-07-13 02:53):
tests: re-add module argument

These were removed as part of dead code removal, but they are actually
needed in the integration tests for comparing with the configuration
generated by the installation.

Hunk 1 of 1 @@ -0 +1,2 @@
+{ config, pkgs, ... }:
+